zerver/openapi/zulip.yaml

```diff
@@ -1206,25 +1266,1581 @@ paths:
                                     message_id:
                                       type: integer
                                       description: |
-                                        The id of the message from which the reaction was
-                                        removed.
-                                - example:
+                                        The id of the message from which the reaction was
+                                        removed.
+                                - example:
+                                    {
+                                      "type": "reaction",
+                                      "op": "remove",
+                                      "user_id": 10,
+                                      "user":
+                                        {
+                                          "user_id": 10,
+                                          "email": "user10@zulip.testserver",
+                                          "full_name": "King Hamlet",
+                                        },
+                                      "message_id": 52,
+                                      "emoji_name": "tada",
+                                      "emoji_code": "1f389",
+                                      "reaction_type": "unicode_emoji",
+                                      "id": 0,
+                                    }
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent when an attachment is sent in a Zulip message.
+                                Sent to all users who received the message with the
+                                attachment.
```

Comment:
These descriptions are wrong -- it's only sent to the user who uploaded them.  You can easily check this by reading the `send_event` arguments in `actions.py` -- which `user_profile` IDs are passed in the third argument determines who receives the event.